### ISSUE TITLE
[BUG] [chromium] Using `ReadableStream` as a body in `fetch` won't populate `request.postData()` of `page.route(...)` handler

### ISSUE THREAD

**Issue by ZitRos** [microsoft/playwright#22594]
TL;DR this client fetch code (you can try in the real browser by opening DevTools at origin [postman-echo.com/get](https://postman-echo.com/get) and pasting):

```typescript
  const url = 'https://postman-echo.com/post';
  const stream = new ReadableStream({
    start: (controller) => {
      controller.enqueue('chunk1');
      controller.enqueue('chunk2');
      controller.close();
    },
  }).pipeThrough(new TextEncoderStream());
  const res = await fetch(url, {
    method: 'POST',
    body: stream,
    duplex: 'half'
  });
  console.log(await res.json())
```

Returns the following response in Chrome:

<img width="540" alt="image" src="https://user-images.githubusercontent.com/4989256/233991544-dccc4ec1-3e37-47de-a9e6-0a0bab51e3a7.png">

But it seems that it's currently impossible to read this request **body** when handling request using Playwright's mocks:

```typescript
page.route('**', (route, request) => {
  console.log(request.postData()); // is always null
  route.fulfill({});
})
```

### System info

- Playwright Version: 1.32.0
- Operating System: macOS Ventura 13.2.1
- Browser: latest auto-downloaded WebKit
- Other info: node v19.6.0

### Source code

For demonstration, `test/dummy.test.ts`:

```ts
import { expect, test } from '@playwright/test';

const RESOURCE = 'https://postman-echo.com/post';

test('dummy', async ({ page }) => {
  let requestBody: any = '';
  await page.route(RESOURCE, async (route, request) => {
    requestBody = request.postData();
    console.log('!!! Request body:', requestBody);
    await route.fulfill({ contentType: 'text/plain', body: 'intercepted' });
  });
  await page.on('console', (msg) => console.log(msg));
  await page.goto(RESOURCE); // Just go there to have the same origin

  await page.evaluate(
    async ([RESOURCE]) => {
      const stream = new ReadableStream({
        start: (controller) => {
          controller.enqueue('chunk1');
          controller.enqueue('chunk2');
          controller.close();
        },
      }).pipeThrough(new TextEncoderStream());
      const res = await fetch(RESOURCE, {
        method: 'POST',
        body: stream,
        ...{ duplex: 'half' },
      });
      console.log('Request response:', await res.text());
    },
    [RESOURCE]
  );

  expect(requestBody).toBe('chunk1chunk2');
});
```

<details>
  <summary>playwright.config.ts</summary>
  
  ```typescript
  import { defineConfig, devices } from '@playwright/test';

  export default defineConfig({
    testDir: 'test',
    fullyParallel: false,
    retries: 0,
    workers: 1,
    use: {
      baseURL: 'http://x.com',
      trace: 'on-first-retry',
    },
    projects: [
      {
        name: 'chromium',
        use: { ...devices['Desktop Chrome'] },
      },
    ],
  });

  ```
  
</details>

Run the test:

```sh
npx playwright install
npx playwright test
```

**Expected**

The request body is `"chunk1chunk2"`.

**Actual**

The request body is always `null`.

# THE GOAL

Being able to read request body given as `ReadableStream` in Playwright's [intercepted network request](https://playwright.dev/docs/network).

**Comment by pavelfeldman** [comment]
Debug note, that the only request event I'm getting when interception is on.

```js
{
  "method": "Network.requestWillBeSent",
  "params": {
    "requestId": "829821.2",
    "loaderId": "286A02780FDB4AAD093D96F3C42120A8",
    "documentURL": "https://postman-echo.com/",
    "request": {
      "url": "https://postman-echo.com/post",
      "method": "POST",
      "headers": {
        "sec-ch-ua": "\"HeadlessChrome\";v=\"113\", \"Chromium\";v=\"113\", \"Not-A.Brand\";v=\"24\"",
        "Referer": "https://postman-echo.com/",
        "Accept-Language": "en-US",
        "sec-ch-ua-mobile": "?0",
        "User-Agent": "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) HeadlessChrome/113.0.5672.53 Safari/537.36",
        "sec-ch-ua-platform": "\"Linux\""
      },
      "mixedContentType": "none",
      "initialPriority": "High",
      "referrerPolicy": "strict-origin-when-cross-origin",
      "isSameSite": true
    },
    "timestamp": 80030.823934,
    "wallTime": 1682385183.217482,
    "initiator": {
      "type": "script",
      "stack": {
        "callFrames": [
          {
            "functionName": "eval",
            "scriptId": "10",
            "url": "",
            "lineNumber": 8,
            "columnNumber": 22
          },
          {
            "functionName": "evaluate",
            "scriptId": "4",
            "url": "",
            "lineNumber": 198,
            "columnNumber": 16
          },
          {
            "functionName": "",
            "scriptId": "9",
            "url": "",
            "lineNumber": 0,
            "columnNumber": 43
          }
        ]
      }
    },
    "redirectHasExtraInfo": false,
    "type": "Fetch",
    "frameId": "32EC89D3FFBE02188E0E94073CE540B6",
    "hasUserGesture": true
  },
  "sessionId": "F7C5D78536F10DC114068E6016BE8C09"
}
```
